app/views/documents/_attachment.html.erb

```diff
@@ -70,11 +71,11 @@
     <% end %>
 
     <% unless attachment.accessible? %>
-      <div class="accessibility-warning js-toggle-accessibility-warning" id="<%= help_block_id %>">
+      <div data-module="toggle" class="accessibility-warning" id="<%= help_block_id %>">
```

Comment:
Yep, thats [still a hook for styling](https://github.com/alphagov/whitehall/blob/update-attachment-markup/app/assets/stylesheets/frontend/helpers/_attachment.scss#L103)